metals/src/main/scala/scala/meta/internal/metals/BloopServers.scala

```diff
@@ -1,15 +1,17 @@
 package scala.meta.internal.metals
 
-import java.io.ByteArrayInputStream
-import java.io.OutputStream
-import java.io.PrintStream
+import java.io._
```

Comment:
Could we revert this? We usually explicitely specify all the imports and avoid wildcard ones.

Reply:
It is done automatically be either Scalafix or Scalafmt
So I’m afraid after reverting it, it would again turn back to the previous state, perhaps? any suggestion?

Reply:
@ckipp01  I need help with this!

Reply:
Looks like it was actually done by Intellij. Just replace it back with
```
import java.io.ByteArrayInputStream
import java.io.OutputStream
import java.io.PrintStream
```

Reply:
You should be able to just change it to the code snippet I have above. Scalafix won't replace it.

Reply:
> Looks like it was actually done by Intellij. 😱 Hacking on Metals with IntelliJ should be a crime 🚨 .

_Joking of course_